pkg/core/cmdutil/flagutil/flagset.go

```diff
@@ -11,7 +11,7 @@ import (
 )
 
 var (
-	ValidOutputFormats = []string{dump.JSONFormat, dump.YAMLFormat, dump.YMLFormat}
+	ValidOutputFormats = []string{dump.JSONFormat, dump.YAMLFormat, dump.YMLFormat, dump.EmptyFormat}
```

Comment:
Nice! That was area I was expecting you to have problem with. You went ahead and resolved it. 
Basically this will allow users to suspend output on other commands than topic create.